### PR TITLE
[TFLite] Support quantised SQUARED_DIFFERENCE operator

### PR DESCRIPTION
**Pull request by leandron**
Add support and test to the SQUARED_DIFFERENCE operator in the TFLite frontend.

Co-Authored-By: Shai Maor <Shai.Maor@arm.com>

cc @ekalda @NicolaLancellotti @Mousius @manupa-arm @areusch

### CODE REVIEW

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -1370,11 +1378,7 @@ def convert_greater(self, op):
     def convert_squared_difference(self, op):
         """Convert TFLite SQUARED DIFFERENCE"""
         # Check if the input tensor is quantized, call QNN op
-        if self.is_quantized(op):
-            raise tvm.error.OpNotImplemented(
-                "TFlite quantized squared difference operator is not supported yet."
-            )
-        difference = self._convert_elemwise(_op.subtract, op)
+        difference = self._convert_elemwise(_op.subtract, op, dequantize=True)
```
Comment (NicolaLancellotti) [inline python/tvm/relay/frontend/tflite.py:1381]: Shouldn't there be a check like the following?

```suggestion
        difference = self._convert_elemwise(_op.subtract, op, dequantize=self.is_quantized(op))
```

File: tests/python/frontend/tflite/test_forward.py [hunk 2]
```diff
@@ -286,6 +287,7 @@ def compare_tflite_with_tvm(
         # convert to tflite model
         converter = tf.lite.TFLiteConverter.from_session(sess, input_tensors, output_tensors)
         converter.experimental_new_converter = experimental_new_converter
+        converter.experimental_new_quantizer = experimental_new_quantizer
```
Comment (NicolaLancellotti) [inline tests/python/frontend/tflite/test_forward.py:290]: Duplicate line.

File: tests/python/frontend/tflite/test_forward.py [hunk 3]
```diff
@@ -270,6 +270,7 @@ def compare_tflite_with_tvm(
     input_range=None,
     mode="graph_executor",
     experimental_new_converter=False,
+    experimental_new_quantizer=False,
```
Comment (NicolaLancellotti) [inline tests/python/frontend/tflite/test_forward.py:273]: Duplicate line.

File: tests/python/frontend/tflite/test_forward.py [hunk 4]
```diff
@@ -2135,6 +2138,7 @@ def __test_elemwise(in_data):
                 quantized=True,
                 input_range=input_range,
                 experimental_new_converter=same_qnn_params,
+                experimental_new_quantizer=experimental_new_quantizer,
```
Comment (NicolaLancellotti) [inline tests/python/frontend/tflite/test_forward.py:2141]: I think the previous line should be removed.

**Comment by leandron** [comment]
Closing in favour of https://github.com/apache/tvm/pull/14667

**Pull request closed without merging**